### PR TITLE
Moving All handling of broker service from EA.D into the implementation IVisualDiagnosticsLanguageService

### PR DESCRIPTION
**Pull request by chabiss**
The Roslyn team have asked us to move all handling of broker services from EA.D into IVisualDiagnosticsLanguageService. This effectively reduces IVisualDiagnosticsLanguageService to one method "InitializeAsync", which initializes IVisualDiagnosticsLanguageService and cause the implementation to deal directly with the Broker service for starting up Hot Reload

### CODE REVIEW

**Review by BretJohnson — APPROVED** [review]

File: src/Tools/ExternalAccess/VisualDiagnostics/Contracts/IVisualDiagnosticsServiceBroker.cs [hunk 1]
```diff
@@ -9,13 +9,10 @@
 namespace Microsoft.CodeAnalysis.ExternalAccess.VisualDiagnostics.Contracts
 {
     /// <summary>
-    /// Facade interface for getting various service brokers
+    /// Facade interface for getting broker service successfully through MEF
     /// </summary>
     internal interface IVisualDiagnosticsBrokeredDebuggerServices
```
Comment (BretJohnson) [inline src/Tools/ExternalAccess/VisualDiagnostics/Contracts/IVisualDiagnosticsServiceBroker.cs:13]: Did you intended to also rename this interface to IVisualDiagnosticsServiceBroker? The type name no longer matches the source file name and the source file name seems better.

File: src/Tools/ExternalAccess/VisualDiagnostics/Internal/VisualDiagnosticsServiceBroker.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+using Microsoft.VisualStudio.Composition;
+using Microsoft.VisualStudio.Shell.ServiceBroker;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.VisualDiagnostics.Internal
+{
+    /// <summary>
+    /// This is a simple wrapper to succeed at getting the broker service using System.ComponentModel.Composition inside an LSP service OnInitialized factory
+    /// </summary>
+    [Export(typeof(IVisualDiagnosticsBrokeredDebuggerServices))]
+    internal sealed class VisualDiagnosticsBrokeredDebuggerServices : IVisualDiagnosticsBrokeredDebuggerServices
```
Comment (BretJohnson) [inline src/Tools/ExternalAccess/VisualDiagnostics/Internal/VisualDiagnosticsServiceBroker.cs:20]: Should the class here also be renamed to VisualDiagnosticsServiceBroker, to match the source file name?

File: src/Tools/ExternalAccess/VisualDiagnostics/Contracts/IVisualDiagnosticsServiceBroker.cs (hunk 1 above)
Comment (chabiss) [reply src/Tools/ExternalAccess/VisualDiagnostics/Contracts/IVisualDiagnosticsServiceBroker.cs:13]: Yes, forgot to rename the class.  Thanks!

File: src/Tools/ExternalAccess/VisualDiagnostics/Internal/VisualDiagnosticsServiceBroker.cs (hunk 2 above)
Comment (chabiss) [reply src/Tools/ExternalAccess/VisualDiagnostics/Internal/VisualDiagnosticsServiceBroker.cs:20]: Yes

**Pull request merged**